src/components/Econtext/components/EcontextInput/EcontextInput.tsx

```diff
@@ -1,21 +1,24 @@
-import { h } from 'preact';
-import { useEffect, useRef, useState } from 'preact/hooks';
+import {h} from 'preact';
```

Comment:
Mmm it seems your Prettier plugin is not picking up the `prettier.config.js` file.

Reply:
Hmm. Will check it out :)

Reply:
Fixed